assets/sass/v2/_beacon.scss

```diff
@@ -0,0 +1,17 @@
+/* stylelint-disable declaration-no-important */
+.okta-sign-in-header {
+  padding-bottom: 90px !important;
```

Comment:
this seems change existing V1 styles. I managed to hack it like this

```
diff --git a/assets/sass/v2/_beacon.scss b/assets/sass/v2/_beacon.scss
index e150b6fc..1621dae9 100644
--- a/assets/sass/v2/_beacon.scss
+++ b/assets/sass/v2/_beacon.scss
@@ -1,14 +1,7 @@
-/* stylelint-disable declaration-no-important */
-.okta-sign-in-header {
-  padding-bottom: 90px !important;
-}
-.siw-main-body {
-  padding-top: 60px;
-}
 .siw-main-header {
   position: relative;
   .beacon-container {
-    top: -60px;
+    top: -160px;
     z-index: 10;
     .auth-beacon-factor {
       cursor: initial;
diff --git a/src/v2/view-builder/internals/HeaderView.js b/src/v2/view-builder/internals/HeaderView.js
index bed296e8..08178d84 100644
--- a/src/v2/view-builder/internals/HeaderView.js
+++ b/src/v2/view-builder/internals/HeaderView.js
@@ -1,4 +1,4 @@
-import { View } from 'okta';
+import { View, $ } from 'okta';
 import FactorBeacon from '../components/FactorBeacon';
 import Animations from 'util/Animations';
 
@@ -9,6 +9,8 @@ export default View.extend({
   },
 
   postRender () {
+    $('#okta-sign-in').removeClass('no-beacon');
+
     // animate beacon
     var selector = '[data-type="beacon-container"]', container = this.$el.find(selector);
     Animations.explode(container);
```

Reply:
This was done to override the styles coming from the okta theme file, which gets imported after the v2/beacon.sass from what I checked.

Reply:
I'll try what you have above

Reply:
we dont have flag into CSS. the override will impact across the board.